src/Microsoft.OData.Client/DataServiceClientFormat.cs

```diff
@@ -232,7 +232,7 @@ internal static void ValidateCanReadResponseFormat(IODataResponseMessage respons
         /// <returns>A service model to be used in format tracking</returns>
         internal IEdmModel LoadServiceModelFromNetwork()
         {
-            HttpWebRequestMessage httpRequest;
+            HttpClientRequestMessage httpRequest;

```

Comment:
Could this work better if we used the interface type (`IODataRequestMessage`) instead of the class type?

Reply:
resolved